packages/core/admin/admin/src/content-manager/components/DynamicTable/TableRows/index.js

```diff
@@ -41,9 +41,22 @@ const TableRows = ({
         );
 
         return (
-          <Tr key={data.id}>
+          <Tr
```

Comment:
It seems that we cannot click on the relation link neither on the local one.

Can you check I'll review it back after it is fixed.

Also can you update your branch?